src/site/xdoc/server/config-ssl-tls.xml

```diff
@@ -109,6 +109,29 @@
 &lt;/tls&gt;
 </source>
 
+      <p>When you enable TLS, you may also configure the server to request a client certificate for authentication:</p>
```

Comment:
Idem a distinct sub-section could be nice.

Reply:
sure thing.